src/lib/shorten/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+
+package(default_visibility = ["//:__subpackages__"])
+
+ts_library(
+    name = "shorten",
+    srcs = ["shorten.ts"],
+    module_name = "@bazel/shorten",
```

Comment:
A bit confusing using `@bazel/shorten` at the import site. Better name?

Reply:
Oh. Was the intention to also publish to @bazel/shorten?

Reply:
yes, I think we should publish some dummy example library so we demonstrate how you'd share some code with a non-bazel part of your org.